library/src/main/java/com/pokegoapi/api/inventory/Inventories.java

```diff
@@ -33,13 +33,13 @@
 import com.pokegoapi.exceptions.LoginFailedException;
 import com.pokegoapi.exceptions.RemoteServerException;
 import com.pokegoapi.main.ServerRequest;
+import com.pokegoapi.api.player.Stats;
```

Comment:
The line 150 doesn't figure because I haven't changed it, I only added the import that tells that the class Stats at line 150 isn't more the one in the same package but the one in com.pokegoapi.api.player.